app/store/services/activities.js

```diff
@@ -221,30 +221,30 @@ export const computePerformance = (activities = {}) => {
       details: [
         {
           name: "distance",
-          percent: Math.trunc(distanceHeuristic * 100 / performance),
-          value: Math.trunc(overallPerformance.distance / 1000),
+          percent: distanceHeuristic,
+          value: overallPerformance.distance,
```

Comment:
was units wrong?

Reply:
all of that should be handled by view - selector! 
psycho-rigid' styl'

Reply:
ok so why didnt you change units?

Reply:
don't get your point! you want me to get ride of unit? 
ps: I was thinking about implementing a SI unit sytem to fix this!